plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -422,6 +422,20 @@ public function admin_scripts() {
 					'i18n_delete_note'                   => __( 'Are you sure you wish to delete this note? This action cannot be undone.', 'woocommerce' ),
 					'i18n_apply_coupon'                  => __( 'Enter a coupon code to apply. Discounts are applied to line totals, before taxes.', 'woocommerce' ),
 					'i18n_add_fee'                       => __( 'Enter a fixed amount or percentage to apply as a fee.', 'woocommerce' ),
+					'i18n_product_gpt_title'             => __( 'Write a description', 'woocommerce' ),
+					'i18n_product_gpt_input_title'       => __( 'ABOUT YOUR PRODUCT', 'woocommerce' ),
+					'i18n_product_gpt_input_placeholder' => __( 'e.g. organic and sustainable skin cleanser for women', 'woocommerce' ),
+					'i18n_product_gpt_buttons_title'     => __( 'TONE OF VOICE', 'woocommerce' ),
+					'i18n_product_gpt_button_casual'     => __( 'Casual', 'woocommerce' ),
+					'i18n_product_gpt_button_formal'     => __( 'Formal', 'woocommerce' ),
+					'i18n_product_gpt_button_flowery'     => __( 'Flowery', 'woocommerce' ),
+					'i18n_product_gpt_button_convincing' => __( 'Convincing', 'woocommerce' ),
+					'i18n_product_gpt_button_accept'     => __( 'Accept', 'woocommerce' ),
```

Comment:
I think "Accept" feels vague here. Can you replace with "Write description", like in Jarek's design?